doc/reference/peripherals/ps2.rst

```diff
@@ -0,0 +1,21 @@
+.. _ps2:
+
+
+PS2
```

Comment:
Technically, the name is ``PS/2`` (while the Playstation 2 is abbreviated PS2) so change mentions of PS2 in the text to be PS/2